pkg/trace/agent/agent.go

```diff
@@ -296,17 +296,27 @@ func (a *Agent) convertStats(in pb.ClientStatsPayload, lang, tracerVersion strin
 	in.Lang = lang
 	for i, group := range in.Stats {
 		n := 0
+		updated := false
```

Comment:
nit: but we usually write
```suggestion
		var changed bool
```
Seems like `changed` may be a better word?